common/src/app/common/types/container.cljc

```diff
@@ -302,13 +302,21 @@
          objects         (:objects container)
          unames          (volatile! (cfh/get-used-names objects))
 
+         component_children
+         (d/index-by :id (cfh/get-children-with-self objects (:id component-shape)))
+
+         ;; We must to avoid that destiny frame is inside the component frame
```

Comment:
"Destiny" is quite a too dramatic word for this, isn't it? :laughing:   Perhaps better "destination"...